src/app/extractor.js

```diff
@@ -123,7 +123,7 @@ export const extractData = async (zip) => {
     console.log(`[debug] ${extractedData.channels.length} channels loaded.`);
 
     const words = extractedData.channels.map((channel) => channel.messages).flat().map((message) => message.words).flat().filter((w) => w.length > 5);
-    extractedData.favoriteWord = mostOccurences(words);
+    extractedData.favoriteWord = mostOccurences(words, true).filter((word) => !['!', '?', '/', '<', '@'].includes(word[0])).slice(0, 10).join(', ');
```

Comment:
I think that it should be an array `favoriteWords`. The `join()` should be done on the main page with Svelte

Reply:
Ok, I will change that.